src/pages/Fox/components/FoxWifHatHoldingLine.tsx

```diff
@@ -0,0 +1,87 @@
+import type { StackDirection } from '@chakra-ui/react'
+import { Button, Flex, HStack, Stack, Text, useColorModeValue } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { WalletIcon } from 'components/Icons/WalletIcon'
+import { middleEllipsis } from 'lib/utils'
+
+type FoxWifHatHoldingLineProps = {
+  accountId: string
+  accountNumber: number
+  amount: string
```

Comment:
suggestion: add unit (i.e `cryptoPrecision`) vernacular

Ideally, we'd make this `amountCryptoBaseUnit` and pass base unit amount as these are always prefered.